helm/app-operator/templates/configmap.yaml

```diff
@@ -21,7 +21,7 @@ data:
         {{- if hasSuffix "-unique" .Release.Name }}
         webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ include "resource.default.namespace" . }}.svc.cluster.local:{{ .Values.port }}'
         {{- else }}
-        webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}'
+        webhookBaseURL: 'https://{{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}'
```

Comment:
It should be HTTPS, of course. I also test https schema with a service object, but it didn't work.